### DISCUSSION TITLE
Searching all visible files

### DISCUSSION THREAD

**Discussion by choskeli** [nvim-neo-tree/neo-tree.nvim#1431] [General]
Is there a way to search all visible files? When I make dotfiles visible the '/' key still doesn't include them in searches.

**Comment by pysan3** [comment]
Please modify `find_args` config to your liking. 

https://github.com/nvim-neo-tree/neo-tree.nvim/blob/660427550bcb75ff161e65695190f7760ae951e0/lua/neo-tree/defaults.lua#L491

Unfortunately the state of neo-tree (whether hidden is shown or not) is not correlated to the search result. (However that's quite interesting and would be worth a feature request IMO).

**Reply by choskeli** [reply]
Hmm I must be misunderstanding. Is this how it would look for lazyvim? It isn't working for me.

```
return {
  "nvim-neo-tree/neo-tree.nvim",
  opts = {
    filesystem = {
      filtered_items = {
        visible = true,
      },
    },
    find_args = function(cmd, path, search_term, args)
      table.insert(args, "--hidden")
      table.insert(args, "--exclude")
      table.insert(args, ".git")
      return args
    end,
  },
}
```

EDIT: I've submitted https://github.com/nvim-neo-tree/neo-tree.nvim/issues/1432

**Reply by pysan3** [reply]
You will want to specify the `find_cmd` (my bad I forgot the actual option name) or your search method may not understand the args. 

Those args in the example are for `fd` which is an alternative to the Linux find command written in rust. If you only have find command installed the option would be different and please read `man find`.
